psqlextra/backend/schema.py

```diff
@@ -63,6 +47,10 @@ def __init__(self, connection, collect_sql=False, atomic=True):
         self.deferred_sql = []
         self.introspection = PostgresIntrospection(self.connection)
 
+    def execute(self, sql, params=()):
+        """execute query"""
+        return super().execute(sql, params)
```

Comment:
Why is this here?

Reply:
that could actually be removed.
I added that to print sql statement and debug my code implementation.